### PR TITLE
add ORGANIZATION.md

### PR DESCRIPTION
**Pull request by chrisplatorres**
This PR proposes adding a ORGANIZATION.md that outlines the directories in the Exokit repo.

--------------------------------------------

Inspired by A-Frame project's [ORGANIZATION.md](https://github.com/aframevr/aframe/blob/master/ORGANIZATION.md).

### CODE REVIEW

File: ORGANIZATION.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    - `util` - Exokit utility native source and header files.
+    - `videocontext` - FFMPEG bindings native source and header files.
+    - `webaudiocontext` - LabSound bindings native source and header files.
+    - `webglcontext` - WebGL bindings native source and header files.
+    - `webrtc` - WebRTC bindings native source and header files.
+    - `windowsystem` - Window bindings native source and header files.
+  - `openvr` - OpenVR native source and header files.
+- `docs` - Complete documentation in Markdown.
+- `examples` - WebXR Examples.
+- `lib` - Three.js minified.
```
Comment (modulesio) [inline ORGANIZATION.md]: It's a custom build with only the THREE.js math parts. It's not full THREE.js, which we don't need.

File: ORGANIZATION.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+- `metadata` - Target-specific app metadata.
+- `res` - App resources for Android-based targets.
```
Comment (modulesio) [inline ORGANIZATION.md]: It's for ML actually.

File: ORGANIZATION.md [hunk 3]
```diff
@@ -0,0 +1,30 @@
+# Directory Structure
+
+- `assets` - Assets used in Exokit builds.
+- `buildtools` - Build tools used in Windows Exokit builds.
+- `deps` - Native dependency source and header files.
+  - `exokit-bindings` - Native exokit bindings.
+    - `bindings` - Exokit bindings native source and header files.
+    - `browser` - Servo2D native source and header files.
```
Comment (modulesio) [inline ORGANIZATION.md]: It's actually CEF primarily, and we're probably deleting Servo.

File: ORGANIZATION.md [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+    - `canvas` - Canvas native source and header files.
+    - `canvascontext` - Canvas Context native source and header files.
+    - `egl` - EGL bindings native source and header files.
+    - `glfw` - GLFW bindings native source and header files.
+    - `gluegen` - GlueGen bindings native source and header files.
```
Comment (modulesio) [inline ORGANIZATION.md]: Soon to be deleted.

File: ORGANIZATION.md [hunk 5]
```diff
@@ -0,0 +1,30 @@
...
+    - `magicleap` - Magic Leap bindings native source and header files.
+    - `util` - Exokit utility native source and header files.
+    - `videocontext` - FFMPEG bindings native source and header files.
+    - `webaudiocontext` - LabSound bindings native source and header files.
+    - `webglcontext` - WebGL bindings native source and header files.
+    - `webrtc` - WebRTC bindings native source and header files.
+    - `windowsystem` - Window bindings native source and header files.
+  - `openvr` - OpenVR native source and header files.
+- `docs` - Complete documentation in Markdown.
+- `examples` - WebXR Examples.
```
Comment (modulesio) [inline ORGANIZATION.md]: 2D and WebVR, too.

File: ORGANIZATION.md (hunk 1 above)
Comment (modulesio) [inline ORGANIZATION.md]: It's a custom build with only the THREE.js math parts. It's not full THREE.js, which we don't need.

**Comment by chrisplatorres** [comment]
Closing in favor of #985.

**Pull request closed without merging**